packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -206,12 +206,30 @@ export default class Editor {
      * Replace a node in editor content with another node
      * @param existingNode The existing node to be replaced
      * @param new node to replace to
+     * @param darkMode (optional) Whether to transform new node to dark mode. Default is false
      * @returns true if node is replaced. Otherwise false
      */
-    public replaceNode(existingNode: Node, toNode: Node): boolean {
+    public replaceNode(existingNode: Node, toNode: Node, darkMode?: boolean): boolean {

```

Comment:
suggest, rename "darkMode" to "transformColorForDarkMode"

Reply:
Updated. Thanks!